beetsplug/spotify.py

```diff
@@ -536,3 +556,71 @@ def _output_match_results(self, results):
             self._log.warning(
                 f'No {self.data_source} tracks found from beets query'
             )
+
+    def _fetch_info(self, items, write, force):
+        """Obtain track information from Spotify."""
+        spotify_audio_features = {
+            'acousticness': ['spotify_track_acousticness'],
+            'danceability': ['spotify_track_danceability'],
+            'energy': ['spotify_track_energy'],
+            'instrumentalness': ['spotify_track_instrumentalness'],
+            'key': ['spotify_track_key'],
+            'liveness': ['spotify_track_liveness'],
+            'loudness': ['spotify_track_loudness'],
+            'mode': ['spotify_track_mode'],
+            'speechiness': ['spotify_track_speechiness'],
+            'tempo': ['spotify_track_tempo'],
+            'time_signature': ['spotify_track_time_sig'],
+            'valence': ['spotify_track_valence'],
+        }
```

Comment:
Maybe this should be a module-level constant?

Also, I'm not 100% clear what he values here are lists instead of single strings. Is the idea that there may be multiple mappings per attribute, someday?

Reply:
I guess one other option would be that we could just make _all_ of these `spotify_X` where `X` is the original Spotify name. That could be a little more predictable and would not require an explicit mapping.

Reply:
Changed